### PR TITLE
INTYGFV-16599: Update fk7210 with new texts, ids and made necessary changes to schematron and tests. Also a fix when transforming sign date to xml.

### PR DESCRIPTION
**Pull request by mww-sopra-steria**

### CODE REVIEW

File: certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/fk7210/CertificateModelFactoryFK7210.java [hunk 1]
```diff
@@ -141,10 +138,10 @@ public CertificateModel create() {
   private static ElementSpecification categoryBeraknatNedkomstdatum(
```
Comment (ChrilleSigerhed) [inline certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/fk7210/CertificateModelFactoryFK7210.java:138]: Should we update the category & question method to Fodelsedatum aswell?

File: certificate-service/app/src/main/java/se/inera/intyg/certificateservice/testability/certificate/service/fillservice/fk7210/TestabilityCertificateFillServiceFK7210.java [hunk 2]
```diff
@@ -34,14 +34,14 @@ public List<ElementData> fill(CertificateModel certificateModel,
 
   private static List<ElementData> fillWithValues(CertificateModel certificateModel) {
     final var elementSpecification = certificateModel.elementSpecification(
-        QUESTION_BERAKNAT_NEDKOMSTDATUM_ID);
+        QUESTION_BERAKNAT_FODELSEDATUM_ID);
     final var elementValue = elementSpecification.configuration().emptyValue();
     if (elementValue instanceof ElementValueDate elementValueDate) {
       final var beraknatNedkomstdatum = elementValueDate
```
Comment (ChrilleSigerhed) [inline certificate-service/app/src/main/java/se/inera/intyg/certificateservice/testability/certificate/service/fillservice/fk7210/TestabilityCertificateFillServiceFK7210.java:40]: Could be updated to beraknatFodelsedatum

File: certificate-service/app/src/main/resources/fk7210/schematron/igrav.v1.sch [hunk 3]
```diff
@@ -14,31 +14,31 @@
...
-      <iso:assert test="count(gn:svar[@id='1']) = 1">
-        Ett intyg måsta ha ett 'Beräknat nedkomstdatum' svar
+      <iso:assert test="count(gn:svar[@id='54']) = 1">
+        Ett intyg måsta ha ett 'Beräknat födelsedatum' svar
       </iso:assert>
     </iso:rule>
   </iso:pattern>
 
   <iso:pattern id="q1">
     <iso:rule context="//gn:svar[@id='1']">
```
Comment (ChrilleSigerhed) [inline certificate-service/app/src/main/resources/fk7210/schematron/igrav.v1.sch:24]: Should this be updated to 54 aswell?

**Review by ChrilleSigerhed — CHANGES_REQUESTED** [review]
Awsome work!

Added some comments to consider regarding minor updates that may or may not be required.

File: certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/fk7210/CertificateModelFactoryFK7210.java (hunk 1 above)
Comment (mww-sopra-steria) [reply certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/fk7210/CertificateModelFactoryFK7210.java]: Fixed. Also did some search and replace in other classes where the old term was used which I had missed.

File: certificate-service/app/src/main/resources/fk7210/schematron/igrav.v1.sch (hunk 3 above)
Comment (mww-sopra-steria) [reply certificate-service/app/src/main/resources/fk7210/schematron/igrav.v1.sch]: Nice catch, was surprised that the tests didn't catch this though. Fixed!

**Review by ChrilleSigerhed — APPROVED** [review]
Looks great!

**Pull request merged**